stories/example-screen-staff-list.stories.js

```diff
@@ -27,150 +17,1023 @@ export default {
+            </svg>
+          </span>
+          <span class="flex-fill">
+            My Account
+          </span>
+        </a>
+      </li>
+    </ul>
+    <hr class="ms-4 me-4 mt-2 mb-2">
+    <ul class="navbar-nav pb-4">
+      <li class="nav-item">
+        <a class="nav-link text-dark ms-4 me-4 mt-10 mb-10 h-auto" href="#">
+          <span>
+            <svg width="1rem" height="1rem" class="me-2 mt-n1">
+              <use xlink:href="${iconSprite}#tabler-home" />
+            </svg>
+          </span>
+          <span class="flex-fill">
+            Help Center
+          </span>
+        </a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark ms-4 me-4 mt-10 mb-10 h-auto" href="#">
+          <span>
+            <svg width="1rem" height="1rem" class="me-2 mt-n1">
+              <use xlink:href="${iconSprite}#tabler-home" />
+            </svg>
+          </span>
+          <span class="flex-fill">
+            System Info
+          </span>
+        </a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark ms-4 me-4 mt-10 mb-10 h-auto" href="#">
+          <span>
+            <svg width="1rem" height="1rem" class="me-2 mt-n1">
+              <use xlink:href="${iconSprite}#tabler-home" />
+            </svg>
+          </span>
+          <span class="flex-fill">
+            Neighborhood Login
+          </span>
+        </a>
+      </li>
+    </ul>
+    <hr class="ms-4 me-4 mt-2 mb-2">
+    <ul class="navbar-nav pb-4">
+      <li class="nav-item">
+        <a class="nav-link text-dark ms-4 me-4 mt-10 mb-10 h-auto" href="#">
+          <span>
+            <svg width="1rem" height="1rem" class="me-2 mt-n1">
+              <use xlink:href="${iconSprite}#tabler-point" />
+            </svg>
+          </span>
+          <span class="flex-fill">
+            Online Staff (2)
+          </span>
+        </a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark ms-4 me-4 mt-10 mb-10 h-auto" href="#">
+          <span>
+            <svg width="1rem" height="1rem" class="me-2 mt-n1">
+              <use xlink:href="${iconSprite}#tabler-point" />
+            </svg>
+          </span>
+          <span class="flex-fill">
+            Online Resident Assistant (2)
+          </span>
+        </a>
+      </li>
+    </ul>
+  </nav>
+`
+
+const staff_list_html = `
+<div class="m-0" style="background-color: ${styles.wrapper.bg};">
+  <div class="mx-auto my-0" style="max-width: ${styles.wrapper.mw}">
+    ${headerLG}
+    ${headerSM}    
+    <div class="d-flex mt-4 w-100">
+      <div class="d-none d-lg-block">
+        ${sidebarNavLG}
+      </div>      
+      <div class="d-flex flex-column justify-content-start flex-wrap w-100 px-2">
+        <div class="d-flex justify-content-between align-items-center flex-wrap px-2 w-100">
+          <h2>Incident Reports</h2> 
+          <button type="button" class="btn btn-sm btn-danger">
```

Comment:
Remove class `btn-sm` here, to match mock more closely (it still won't be perfect).